src/abilities/Snow-Bunny.js

```diff
@@ -342,7 +342,8 @@ export default G => {
 				let dist = projectileInstance[2];
 
 				tween.onComplete.add(function() {
-					ability.destroy();
+					// this refers to the animation object, _not_ the ability
```

Comment:
Capitalized comments.

Hmm, what was the logic of the actual bug btw?

Reply:
javascript has a major `this` problem. i incorrectly changed `this` to `ability` and that caused the bug.

Reply:
To be fair, `this` is only a problem in poorly written code and documented code, this nested callback on top of nested callback on top of jQuery events (which rescope this sometimes but not always) on top of whatever else... 👎

I still maintain we should have delayed, and rewrote the entire stack from the ground up using sensible and sane practices to avoid this nightmare of kludges ;)